### DISCUSSION TITLE
Constant "Verify that the daemon is running" error

### DISCUSSION THREAD

**Discussion by KuuminKochi** [OpenTabletDriver/OpenTabletDriver#2634] [Q&A] [answered]
### Acknowledgements

- [X] I have installed OpenTabletDriver by following its [official installation instructions](https://opentabletdriver.net/).
- [X] I have checked the [OpenTabletDriver Wiki](https://opentabletdriver.net/Wiki) and respective FAQ pages, and my issue was not covered or actually fixed.
- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.

### Steps to Reproduce

On Gentoo:
1. Install OpenTabletDriver
2. Run OpenTabletDriver
3. Wait on the "Connecting to OpenTabletDriver Daemon..." screen
4. After a while, you'll get the error
![image](https://user-images.githubusercontent.com/97841709/217280366-f7111a33-9b4b-4932-9116-8343a1d7e09f.png)

### Expected Behavior

OpenTabletDriver starts up normally without any hassle

### Observed Behavior

"Verify that the daemon is running" error 

### Tablet Device

Huion H950P

### Diagnostic Information

![image](https://user-images.githubusercontent.com/97841709/217281032-fcb1e586-af3e-4839-b259-069e24e764a9.png)

**Comment by X9VoiD** [comment]
I'm not 100% sure how OTD is packaged for Gentoo, but if you're using systemd, enable `opentabletdriver.service`. If it's not there, you may adhoc your own solution by running `otd` on boot of your DE/WM.

**Reply by KuuminKochi** [reply]
I use OpenRC, so there's no `opentabletdriver.service`. Running `otd` doesn't seem to work as well, it spits out `No tablets were detected` in the end

**Reply by jamesbt365** [reply]
What tablet do you have?

**Reply by KuuminKochi** [reply]
Huion H950P

**Comment by gonX** [comment]
Please try again with the latest release version (0.6.3.0). You should start the daemon before starting the UI.

**Comment by yazsq** [comment]
Running `otd-daemon` worked for me. Try running it and then starting the app.

**Comment by jamesbt365** [comment, marked as answer]
you must follow the other inits section of our FAQ:
https://opentabletdriver.net/Wiki/FAQ/Linux#other-inits

There is no systemd service for either init system supported on gentoo, you should contact the maintainer of the package if this is a hard requirement for you, otherwise you can use this [service](https://github.com/OpenTabletDriver/OpenTabletDriver/blob/f14a184fef67a52637ab494497a151044bafd553/eng/linux/Generic/usr/lib/systemd/user/opentabletdriver.service) we provide, I'm not sure if the path for these services are different on gentoo, so you will have to look up the process of installing user services yourself.

If you do go down the route of installing this service, make sure that you install it as a systemd **user** service, not a system one.

[a comment by an automated account is not shown]
